utils/modeler/modeler.go

```diff
@@ -70,13 +79,30 @@ func (m *Modeler) mapToModel(data map[string]string, context string, rv reflect.
 		} else {
 			// We're not nested!
 			var key string
+			prefix := m.prefix
+			if prefix != "" && !strings.HasSuffix(prefix, "/") {
+				prefix = fmt.Sprintf("%s/", prefix)
+			}
 			if context == "" {
-				key = fmt.Sprintf("%s/%s", m.prefix, tagValue)
+				key = fmt.Sprintf("%s%s", prefix, fieldTagValue)
 			} else {
-				key = fmt.Sprintf("%s/%s.%s", m.prefix, context, tagValue)
+				key = fmt.Sprintf("%s%s.%s", prefix, context, fieldTagValue)
 			}
 			stringVal, ok := data[key]
 			if ok {
+				constraintTagValue := rf.Tag.Get(m.constraintTag)
+				if constraintTagValue != "" {
+					constraint := regexp.MustCompile(constraintTagValue)
```

Comment:
Good catch. This is an odd case where while all the regexps are known at build time, they may not get evaluated until an arbitrary point in runtime, so we probably shouldn't use a Must method.

Alternately, we could precompile and cache all of the regexps in an init() and use the Must there. Then at runtime, you'd retrieve the cached copy and use it.